js/app.js

```diff
@@ -31,8 +102,22 @@ do {
         number1 = prompt('Podaj liczbę nr 1');
         number2 = prompt('Podaj liczbę nr 2');
 
-        if(action === '+') {
-            calc.add(number1, number2);
+        switch (action) {
+            case '+':
+                calc.add(number1, number2);
+                break;
+            case '-':
+                calc.subtract(number1, number2);
+                break;
+            case '*':
+                calc.multiply(number1, number2);
+                break;
+            case '/':
+                calc.divide(number1, number2);
+                break;
+            case '^':
+                calc.power(number1, number2);
+                break;
```

Comment:
Ten sposób zapisu powoduje, że 100 operacji będzie miało 100 opcji case/if.
Można spróbować zrobić to inaczej.
Spróbuj się zainspirować tym artykułem: https://devmentor.pl/b/mniej-instrukcji-warunkowych